controllers/vrg_volrep.go

```diff
@@ -210,6 +210,17 @@ func (v *VRGInstance) preparePVCForVRProtection(pvc *corev1.PersistentVolumeClai
 		return !requeue, !skip
 	}
 
+	// if PVC is protected by another VRG, skip this PVC
+	// there might be PVCs that don't have a conflicting owner
+	if pvcVRGOwner, found := pvc.Annotations[pvcVRAnnotationOwnerKey]; found {
+		if pvcVRGOwner != v.instance.Name {
```

Comment:
If a PVC is owned by another instance of VRG, then this is an error.  We should at least log it as such.

Reply:
Ack